docs/manifest_content.md

```diff
@@ -414,6 +422,18 @@ agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.Edp.Ne
 agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.Edp.NetworkWatcherAgentWind<br>ows/\*/\*.log
 agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.NetworkWatcherAgentWindows/<br>\*/\*.txt
 agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.NetworkWatcherAgentWindows/<br>\*/\*.log
+aks | copy | /Windows/System32/winevt/Logs/System.evtx
+aks | copy | /Windows/System32/winevt/Logs/Application.evtx
+aks | copy | /k/\*.log
+aks | copy | /k/\*.err
+aks | copy | /k/azure-vnet.log.\*
+aks | copy | /cni/azure-vnet.log

```

Comment:
It is also helpful to get the cni config file at c:\k\azurecni\netconf\10-azure.conflist